sign-up.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Sweater Weather!</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,700" rel="stylesheet">
+  </head>
+  <body>
+    <nav>
+      <div id="page-title">
+        <a href="index.html">Sweater Weather</a>
+      </div>
+      <div id="sign-up-link">
+        <a href="sign-up.html">Sign Up</a>
+      </div>
+    </nav>
+    <div class="errors"></div>
+    <div class="alerts"></div>
+    <div class="container">
+      <form id="sign-up">
+        <h2>Create a New Account</h2>
+        <ul>
```

Comment:
I think using an unordered list in your form here isn't necessary and might make styling more of a pain than it needs to be. The labels and inputs should sit nicely the way you want them without the list.

Reply:
You're totally right! Yay simplifying things!